libra_swarm/src/swarm.rs

```diff
@@ -490,12 +505,28 @@ impl LibraSwarm {
 
     /// Vector with the public AC ports of the validators.
     pub fn get_validators_public_ports(&self) -> Vec<u16> {
```

Comment:
I think you can retire both `get_validators_public_ports` and `get_full_nodes_public_ports` methods and just hide that logic inside of `get_ac_port`. Those methods don't have any other usage